concepticondata/conceptlists/Kibrik-2012-122.tsv

```diff
@@ -96,7 +96,7 @@ Kibrik-2012-122-94	94	run	бежать	1519	RUN	Motion	Activities	Activities not
 Kibrik-2012-122-95	95	cry	плакать	1839	CRY	Emotion	Activities	Activities not involving objects
 Kibrik-2012-122-96	96	sing	петь	1261	SING	Speech/sound	Activities	Activities not involving objects
 Kibrik-2012-122-97	97	sleep	спать	1585	SLEEP	Physiology	Activities	Activities not involving objects
-Kibrik-2012-122-98	98	scream	кричать	2522	SCREAMING	Speech/sound	Activities	Activities not involving objects
+Kibrik-2012-122-98	98	scream	кричать			Speech/sound	Activities	Activities not involving objects
```

Comment:
The gloss here is the one representing the action. It is also reflected in the Russian lexeme. Moreover, Kibrik's list contains verbal concepts only.

Reply:
I see, thanks. So the gloss in Concepticon itself is not ideal? (should be SCREAM instead of SCREAMING?)

Reply:
SCREAM (`Action/Process`) is potentially a good candidate for adding to Concepticon, I'll make a new PR for it. SCREAMING (`Person/Thing`) (i.e. a sharp piercing cry) is perfectly fine for substantive concepts.

Reply:
Well, SCREAMING is just the present participle of `to scream` so it is rather a question of SCREAM (VERB) and SCREAM (NOUN). I think having SCREAMING as the noun is confusing.

Reply:
(not that we care all to much about the accuracy of glosses in some cases, but I don't think having an inflected verb form for a `Person/Thing` category that denotes a noun concept is all that ideal)

Reply:
The two glosses currently mapped to SCREAMING are nouns. One solution could be to rename this concept set to SCREAM (NOISE) (the naming is open to discussion) and add SCREAM (CRY) (?) for verbs.

Reply:
Yes. SCREAM (NOISE) is probably better than SCREAMING.